data/human/human missions.txt

```diff
@@ -7453,3 +7453,442 @@ mission "Adelita 5"
+			`	Howard pauses, gauging your reaction before continuing. "All of the options have their merits, though investors usually prefer the one with the most returns. But the choice is yours if you're in. I'll make sure you get the full details once we're done with this little favor in Oblate."`
+			choice
+				`	"Alright, I'm in. Let's discuss it more after Oblate."`
+					goto accepting
+				`	"Why would a shipyard have refueling stations?"`
+				`	"None of those options sound good to me. I'm out."`
+					decline
+			`	Howard clearly enjoys the opportunity to explain. "Megaparsec's shipyards here on Sunracer are our crown jewel, building nearly every one of our ships as well as our high performance aircraft and hovercraft. You cannot imagine the logistics required to get the countless capacitors and microchips from Tinker, Delta V's plasma engines from Solace - heck, we have a dedicated logistics hangar on Crossroads just for shipping the Quicksilver's Particle Cannons! Don't even get me started on the Mantas! Ada's a long ways away, but we sure love Lovelace Labs' products...`
+			`	"Ahem. All that to say, refueling stations, while few, are part of our larger strategy in vertical integration. Refueling stations are the heart of our logistics network for bringing all the parts we need here and for getting various vehicles out of here to be sold across the Republic.`
+			`	"There are, of course, ancillary benefits as well. For R&D, we can test the impact of different fuel mixtures on our ships prior to entering the market. For partnerships, we can include fueling rights in large-scale sales agreements. At our largest logistics hubs the economies of scale even allow us to competitively offer refueling services on the open market, giving us a diversified revenue stream.`
+			`	"In fact, the ancillary benefits are largely why there's such discretion in how the expansion will go. The aggressive and conservative fueling options both offer R&D and partnership benefits that are more than the project's direct returns."`
+			choice
+				`	"That makes sense. Let's discuss the deal more after Oblate."`
+				`	"Thanks for the overview, but I'm still not interested. I'm out."`
+					decline
+			label accepting
+			`	Howard nods with approval. "Excellent. The Marauder ship was last seen near Oblate, in the outer rim of that system. It's likely to be well armed, so stay sharp. Once you've disabled it, head back here and I'll make sure you're properly compensated."`
+			branch blewupgolden
+				has "syndicate business golden jewel dead"
+			`	He cracks a smile. "I remember the first time we met you left that Vanguard alive. Same deal here, but disable the Marauder. Hopefully that Vanguard's not still around.`
+				goto ending
+			label blewupgolden
+			`	He leans in slightly, lowering his voice. "Just be careful. We don't want to destroy the ship. Cripple it so we can study it, but don't blow it into oblivion like you did to that Vanguard when we first met.`
```

Comment:
So I've got an interesting comment.
Is the existence of the word "to" really justified here?
I think it may be better without it (and I would provide a detailed explanation on why, but I'm not actually sure. I just think it sounds better. Feel free to refute this if you think I'm wrong).
```suggestion
			`	He leans in slightly, lowering his voice. "Just be careful. We don't want to destroy the ship. Cripple it so we can study it, but don't blow it into oblivion like you did that Vanguard when we first met.`
```

Reply:
Yep, that works without "to." Without the "to" is a bit more casual, the "did" just gets an implied object, and the meaning is still clear. Nice phrasing change.